app/views/govuk_publishing_components/components/docs/google_tag_manager_script.yml

```diff
@@ -11,3 +11,7 @@ examples:
   default:
     data:
       gtm_id: GTM-XXXXXXX
+  with_blocklist:
+    data:
+      gtm_id: GTM-XXXXXXX
+      blocklist: "['customPixels', 'customScripts', 'html', 'nonGoogleScripts']"
```

Comment:
Should this be an array?

Reply:
Thanks @kevindew - this is the doc-block, which caused me formatting issues while trying to keep the linter-passing `['customPixels', 'customScripts', 'html', 'nonGoogleScripts']` in the example on-screen. So, yes is should be an array, the issue was trying to make it look like one in the example.